db/migrate/20200806193438_large_table_primary_keys_to_bigints.rb

```diff
@@ -0,0 +1,74 @@
+class LargeTablePrimaryKeysToBigints < ActiveRecord::Migration[6.0]
+  def up
+    puts "ahoy_messages"
```

Comment:
The last migration I ran with custom statements output nothing and it killed me not knowing where we were in the progress. This will allow me to follow along and update the site banner as needed.

Reply:
Not that it matters much, but maybe we can add `if Rails.env.production?` so people don't wonder where this messages come from in their dev environemnts?

Reply:
Better yet, I made them more explicit `migrating ahoy_message PKs to bigints`